randovania/games/samus_returns/exporter/joke_hints.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import annotations
+
+JOKE_HINTS = [
+    "This hint has been sealed away.",
```

Comment:
Got something to indicate that this is a pun, so that it doesnt get mistaken for a bug?

Reply:
It's supposed to be a play on Chozo Seal. I could change it to `The Chozo have Sealed this hint away.`